fabtests/multinode/src/core_coll.c

```diff
@@ -208,11 +220,18 @@ static int sum_all_reduce_test_run()
 		return err;
 	}
 
-	for (i = 0; i < pm_job.num_ranks; i++) {
-		expect_result += i;
+	for (i = av_set_attr.start_addr;
+	     i <= av_set_attr.end_addr;
+	     i += av_set_attr.stride) {
+		expect_result += i + 1;
```

Comment:
I don't understand the +1 here either.

Reply:
data is now set as rank + 1 as described above

Reply:
Maybe add something like:
```
const int base_data_value = 2209; /* any random value here */
```
and use that instead of 1, so that it's clearer what the intent is.

Reply:
Done (and moved to the test PR)